custom_components/moonraker/__init__.py

```diff
@@ -146,12 +146,22 @@ async def async_get_cameras(self):
         """Return list of cameras"""
         return await self._async_fetch_data("server.webcams.list", None)
 
+    async def async_get_printer_object_list(self):
```

Comment:
I think we should create a generic `fetch_data` method.
I don't like that we keep adding logic in the coordinator

(That is more a debate tbh)